docs/spec_draft.md

```diff
@@ -483,6 +484,59 @@ IEEE-754 specification.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.cholesky
+
+### Semantics
+
+Computes the Cholesky decomposition of a batch of symmetric (Hermitian) positive
+definite matrices.
+
+If `lower` is `true`, it computes the lower-triangular matrices `l` such that
+$a = l \cdot l^{T}$. If `lower` is `false`, it computes the upper-triangular
+matrices `u` such that $a = u^{T} \cdot u$.
+
+If the rank of `a` is greater than 2, all except the minor 2 dimensions of `a`
+are treated as batch dimensions.
+
+If `a` is not symmetric (Hermitian) positive definite, the result is
+implementation-defined.
+
+### Inputs
+
+| Name    | Type                                       |
+|---------|--------------------------------------------|
+| `a`     | tensor of floating-point or complex type   |
+| `lower` | 0-dimensional tensor constant of type `i1` |
+
+### Outputs
+
+| Name     | Type                                     |
+|----------|------------------------------------------|
+| `result` | tensor of floating-point or complex type |
+
+### Constraints
+
+  * (C1) `a` and `result` have the same element type.
+  * (C2) rank(`a`) > 2.
```

Comment:
"(C2) `rank(a) >= 2`". 2 is also a valid rank - in that case, there won't be any batch dimensions, but that's fine.

Reply:
Thanks for the catch!